### ISSUE TITLE
Problems with the competitor selection for the leaderboard charts

### ISSUE THREAD

**Issue by aksajhfduwafe** [aksajhfduwafe/bugzilla-to-github-issues-test#299]
| Field | Value |
|---|---|
| **Reporter** | @LennartH |
| **Assignee** | @LennartH |
| **Product** | Sailing Race Analytics |
| **Component** | Frontend |
| **Version** | unspecified |
| **Hardware** | All |
| **OS** | All |
| **Severity** | normal |
| **Priority** | P2 |
| **Created** | 2012-02-16T16:38:17Z |
| **Original Bug** | [Bug 298](https://bugzilla.sapsailing.com/bugzilla/show_bug.cgi?id=298) |

---

Sometimes theres no series displayed in the chart, even if some competitors are selected.

To reconstruct the problem, go in the RaceBoardPanel and select a competitor (the chart gets filled). Then select another competitor (the chart gets filled).
Then select the first competitor again and the chart should be empty.

---
*Resolved as WONTFIX by @axeluhl*

**Comment by aksajhfduwafe** [comment]
**@axeluhl** commented:

Could the remaining problem have to do with the asynchronous callback which somehow interferes with other updates to the chart?

**Comment by aksajhfduwafe** [comment]
**@LennartH** commented:

I don't think so, because the chart had always the correct series after the updateTable().
And the updateTable() is only called from an AsyncCallback.

**Comment by aksajhfduwafe** [comment]
**@axeluhl** commented:

It really looks like a HighCharts bug. I've tried selecting / deselecting. Eventually, I ended up with a situation (in debug mode) where selecting a single competitor let the chart go from no series to three series. In the debugger, however, there was only one call to addSeries that I was able to trace, even when putting the breakpoint into the innermose BaseChart.addSeries(...) method. We should try to raise this as a bug to the HighCharts folks. The problem may be to produce a reproduction scenario for them.
